src/util.rs

```diff
@@ -281,9 +281,16 @@ pub mod utf8 {
 	use libc::c_char;
 
 	pub fn into<'a>(data: *const c_char, length: u32) -> Vec<&'a str> {
-		unsafe {
```

Comment:
I am not sure, as we have to be sure we don't do any of the dereferencing or modifications to `length` if it's already 0. (We really don't want to do anything if `length==0`).

It seemed better to be explicit and say that if `length==0` then we really do nothing with the pointer.